### PR TITLE
Add your own name to copyright part of the page

### PR DESCRIPTION
**Pull request by yigitkeremoktay**
This feature is ready to be deployed. (Edited)

### CODE REVIEW

**Comment by yigitkeremoktay** [comment]
This feature is ready.

**Comment by yigitkeremoktay** [comment]
Fixed a conflict that reverted PR https://github.com/Pryx/server-status/pull/112 since this work was started before it.
Waiting @Pryx s approval.

File: index.php [hunk 1]
```diff
@@ -47,5 +47,6 @@
       </div>
 <?php } 
 
-Template::render_footer();
-}
\ No newline at end of file
+Template::render_footer(false,COPYRIGHT_TEXT);
```
Comment (Pryx) [inline index.php:50]: Passing a named constant seems wrong to me... Why not simply use it in the function directly?

File: template.php [hunk 2]
```diff
@@ -138,7 +138,7 @@ public static function render_footer($admin = false)
     <div id="footerwrap">
       <div class="container">
         <div class="row centered">
-          <div class="col-md-4 text-left"><a href="https://github.com/Pryx/server-status/" target="_blank"><i class="fab fa-github" aria-hidden="true"></i>&nbsp;</a>Copyright © <?php echo date("Y");?> Vojtěch Sajdl</div>
+          <div class="col-md-4 text-left"><a href="https://github.com/Pryx/server-status/graphs/contributors" target="_blank">Copyright © <?php echo date("Y");?> Server Status Project Contributors </a><?php if(strlen($additional_copyright)>1){ echo " and ".$additional_copyright; } ?></div>
```
Comment (Pryx) [inline template.php:141]: Ternary operator and `empty()` seem like a better fit to me, but otherwise it looks great :)

Comment (yigitkeremoktay) [reply template.php:141]: That is because if people defines it as " " (with a space) instead of "" (completely empty).

File: index.php (hunk 1 above)
Comment (yigitkeremoktay) [reply index.php:50]: Will fix

Comment (yigitkeremoktay) [reply index.php]: Fixed

**Comment by yigitkeremoktay** [comment]
Ready to deploy.

**Comment by yigitkeremoktay** [comment]
@Pryx I am waiting for you to approve it so that I can start working on some more features since it becomes really hard to track when there are too may PRs

**Comment by yigitkeremoktay** [comment]
This PR was merged please refer to https://github.com/Pryx/server-status/pull/117

**Pull request closed without merging**